src/pages/EmployeeList/components/EmployeeTable.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { EmployeeRow } from './EmployeeTable/EmployeeRow';
+import { Employee } from '../EmployeeList';
+import styled from 'styled-components';
+
+const Table = styled.table`
+  width: 100%;
+  border-collapse: collapse;
+  border: 2px solid gray;
+`;
+
+const Th = styled.th`
+  background-color: #006999;
+  color: white;
+  text-align: left;
+  padding: 10px;
+`;
+
+const Td = styled.td`
+  padding: 8px;
+  border-bottom: 1px solid #ddd;
+`;
+
+interface EmployeeTableProps {
+  employees: Employee[];
```

Comment:
In this case Employee type would have been more maintainable to be defined locally in this component, even if that means this type is defined multiple times throughout the application.

Reply:
Done.